aiohttp_jinja2/helpers.py

```diff
@@ -0,0 +1,72 @@
+"""
+useful context functions, see
+http://jinja.pocoo.org/docs/dev/api/#jinja2.contextfunction
+"""
+import jinja2
+
+
+@jinja2.contextfunction
+def reverse_url(context, __route_name, **parts):
+    """
+    filter for generating urls, see
```

Comment:
We don't use autodoc (and I reluctant to do it).

From my perspective sphinx markup should be placed in `.rst` files, docstrings should be relative small and don't contain any special symbols: they are invented for helping human reading, not for parsing by automated tool.
The source code reading is not comfortable If docstring is longer than function source itself.

Open CPython source code: it follows the same rule